server/src/setup/db/db.ts

```diff
@@ -0,0 +1,5 @@
+import { Repos } from './repos'
+
+export interface DB {
```

Comment:
Can we combine this file and `./repos.ts` into one file like `./types`? Trying to figure out if splitting everything in their file is worth the cognitive load at this stage of the project.